packages/nextjs/src/utils/serverIntegrations.ts

```diff
@@ -0,0 +1,60 @@
+import { RewriteFrames } from '@sentry/integrations';
+import { Integration } from '@sentry/types';
+
+const SOURCEMAP_FILENAME_REGEX = /^.*\/.next\//;
+
+/** Default RewriteFrames integration to match filenames in Sentry. */
+export const REWRITE_FRAMES_INTEGRATION = new RewriteFrames({
```

Comment:
@kamilogorek do we have a clear preference / naming convention in the SDK code base?

I find the all caps here confusing, I think along the lines of https://stackoverflow.com/a/49537197, but know eventually this is just whatever standard we decide to follow in the repo.

Specifically, I'd expect this to be named `DefaultRewriteFrames` or similar.

Reply:
Agree. This is an instance of integration, not the constant value. I'd go with `defaultRewriteFrames` (lowercase, as it's already instantiated)

Reply:
Since this is an integration, may the usage of "default" generate confusion with [Default Integrations](https://docs.sentry.io/platforms/javascript/configuration/integrations/default/)?

Reply:
I think it should be fine, it's very explicit. `defaultREWRITEFRAMES` :p